libs/@local/harpc/server/src/session.rs

```diff
@@ -78,10 +96,10 @@ impl<T> AsRef<T> for Session<T> {
 
 #[derive(Debug)]
 struct Marked {
-    // we use an std Mutex here, because we do not use the guard across an await point, therefore
-    // an std mutex is faster, smaller and more efficient.
+    // We use a std mutex here, because we do not use the guard across an await point, therefore
+    // a std mutex is faster, smaller and more efficient.
```

Comment:
```suggestion
    // We use an std mutex here, because we do not use the guard across an await point, therefore
    // an std mutex is faster, smaller and more efficient.
```